audio-worklet/bitcrusher-test.html

```diff
@@ -22,16 +22,18 @@
       testIllegalInput();
 
       /**
-       * Populate array with 0:99 and verify the bitcrusher holds an array
-       * where the number of consecutive equal numbers is controlled by the 
-       * variable factor. This verfies the sample rate reduction component
-       * of bitcrusher operating outside of an audio context.
+       * Verify the sample rate reduction component of the bitcrusher operating
+       * outside of an audio context. A bitcrushed array generated from an 
+       * arbitrary sequence of numbers should hold sequences of consecutive 
+       * equal numbers, controlled by the variable factor.
        */
       function testSampleReduction() {
         let context = new OfflineAudioContext(1, 44100, 44100);
 
         for (let reduction = 1; reduction < 10; reduction++) {
-          const bufferSize = 5;
+          // BufferSize chosen arbitrarily and can be any value greater than
+          // reduction to perform the test.
+          const bufferSize = 20;
```

Comment:
It was only broken in the sense that it did not create an array large enough to test a sample reduction of 10. The test would not test what it seemed like it was testing but it wouldn't fail.